tools/calyx-pass/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "calyx-pass"
+default-run = "calyx-pass"
+version = "0.2.1"
+edition.workspace = true
+
+[[bin]]
+name = "calyx-pass"
+path = "src/main.rs"
+
+[dependencies]
+argh = "0.1.12"
```

Comment:
We use `argh` for many of the tools. We should upgrade it to a workspace level crate so that all tools are using the same version.